README.md

```diff
@@ -3,6 +3,11 @@
 
 A community-owned library of policies for the [OPA Gatekeeper project](https://open-policy-agent.github.io/gatekeeper/website/docs/).
 
+## Validation and Mutation
+The library consists of two main components: `Validation` and `Mutation`.
+- Validation: Gatekeeper validates the resources in the cluster against the policies defined in the library. The policies are defined as `ConstraintTemplates` and `Constraints`. The `ConstraintTemplates` define the policy logic and the `Constraints` define the parameters for the policy.
```

Comment:
Something like: "constraint templates" can be applied directly to a cluster. You then apply "constraints" to customize policy to fit your specific needs.

For mutation: "Mutation policies are only examples, they should be customized to meet your needs before being applied"